lib/evss/disability_compensation_form/data_translation_all_claim.rb

```diff
@@ -575,7 +575,10 @@ def greater_itf_date?
 
       def application_create_date
         # Application create date is the date the user began their application
-        @acd ||= InProgressForm.where(form_id: VA526ez::FORM_ID, user_uuid: @user.uuid)
+        # TODO AEC
+        @acd ||= InProgressForm.where(form_id: [FormProfiles::VA526ez::FORM_ID, FormProfiles::VA526ezbdd::FORM_ID],
```

Comment:
This is awkward, but we don't have the InProgressForm id at this point.  Maybe we could just get the FE to send the InProgressForm id on submit.

Reply:
Could you just check the service dates to determine if it's BDD or a normal submission?

Reply:
@SarahJaine had a talk about this yesterday.  I think we're good here.  Most likely, there'll only be one form InProgressForm and in all cases, the one they're submitting will be the most recently edited one.